playbooks/roles/deploy-osh/files/common-images-overrides.yml

```diff
@@ -1,79 +1,20 @@
 images:
   tags:
-    barbican_api: "{{ suse_osh_registry_location }}/openstackhelm/barbican:{{ suse_osh_image_version }}"
-    barbican_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/barbican:{{ suse_osh_image_version }}"
-    bootstrap: "{{ suse_osh_registry_location }}/openstackhelm/heat:{{ suse_osh_image_version }}"
-    cinder_api: "{{ suse_osh_registry_location }}/openstackhelm/cinder:{{ suse_osh_image_version }}"
-    cinder_backup: "{{ suse_osh_registry_location }}/openstackhelm/cinder:{{ suse_osh_image_version }}"
-    cinder_db_sync: "{{ suse_osh_registry_location }}/openstackhelm/cinder:{{ suse_osh_image_version }}"
-    cinder_scheduler: "{{ suse_osh_registry_location }}/openstackhelm/cinder:{{ suse_osh_image_version }}"
-    cinder_volume: "{{ suse_osh_registry_location }}/openstackhelm/cinder:{{ suse_osh_image_version }}"
-    cinder_volume_usage_audit: "{{ suse_osh_registry_location }}/openstackhelm/cinder:{{ suse_osh_image_version }}"
     congress_api: "{{ suse_osh_registry_location }}/openstackhelm/congress:{{ suse_osh_image_version }}"
```

Comment:
This image pointing is also used in airship `site/soc/software/config/versions.yaml` can we unify those in common-vars maybe?

Reply:
I'm not sure. The images I left in common-images-overrides.yml are only those we are not explicitly changing (from upstream helm chart overrides or directly from our socok8s values overrides that did not make it upstream). So I think currently there's no overlap of `common-images-overrides.yml` and ` site/soc/software/config/versions.yaml`

But it's possible I did not get your point.